iteration/iteration.go

```diff
@@ -118,7 +118,9 @@ type GormIterationRepository struct {
 func (m *GormIterationRepository) LoadMultiple(ctx context.Context, ids []uuid.UUID) ([]Iteration, error) {
 	defer goa.MeasureSince([]string{"goa", "db", "iteration", "getmultiple"}, time.Now())
 	var objs []Iteration
-
+	if len(ids) == 0 {
```

Comment:
Add test case

Reply:
added in https://github.com/fabric8-services/fabric8-wit/pull/1865/commits/976f059d14c36e744ce3ee6a11b4203d0b77a119#diff-ac105ac399c839c07e21bac7fbbb8651R442